src/main/java/gov/cabinetofice/gapuserservice/web/LoginController.java

```diff
@@ -44,4 +47,17 @@ public RedirectView redirectOnLogin(@CookieValue(REDIRECT_URL_COOKIE) String red
     private RedirectView redirectToThirdParty() {
         return new RedirectView(authenticationProvider.getUrl()); //TODO make sure COLA's redirect URL points to the domain the user service sits on
     }
+
+    @GetMapping("/validate-user")
+    public ResponseEntity<String> ValidateUser(final @CookieValue(name = USER_SERVICE_COOKIE_NAME, required = false ) String jwt) {
+        if(jwt == null)
+            return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body("Unauthorized");
+
+        //TODO change from colaJwtService to CustomJwtServiceImpl
+        final boolean isJwtValid = colaJwtService.isTokenValid(jwt);
+        if(!isJwtValid)
+            return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body("Unauthorized");
```

Comment:
Could combine these 401 return statements

Reply:
Don't need to return a body either, I suspect if you remove `<String>` from this methods return type then it will be happy